test/unit/price-formatting/format-spec.js

```diff
@@ -676,6 +770,64 @@ describe('price-formatting format', () => {
         expect(parts.Pips).toEqual('');
         expect(parts.DeciPips).toEqual('');
         expect(parts.Post).toEqual('');
+
+        // When supplying DecimalPips
+        parts = priceFormatting.formatPriceParts(123.23456, 4, priceFormatOptions.AllowDecimalPips, null, 2);
+        expect(parts.Pre).toEqual('');
+        expect(parts.First).toEqual('123.23');
+        expect(parts.Pips).toEqual('45');
+        expect(parts.DeciPips).toEqual('60');
+        expect(parts.Post).toEqual('');
+
+        parts = priceFormatting.formatPriceParts(1, 4, priceFormatOptions.AllowDecimalPips, null, 2);
+        expect(parts.Pre).toEqual('');
+        expect(parts.First).toEqual('1.00');
+        expect(parts.Pips).toEqual('00');
+        expect(parts.DeciPips).toEqual('00');
+        expect(parts.Post).toEqual('');
+
+        parts = priceFormatting.formatPriceParts(0.008899, 4, priceFormatOptions.AllowDecimalPips, null, 2);
+        expect(parts.Pre).toEqual('');
+        expect(parts.First).toEqual('0.00');
+        expect(parts.Pips).toEqual('88');
+        expect(parts.DeciPips).toEqual('99');
+        expect(parts.Post).toEqual('');
+
+        parts = priceFormatting.formatPriceParts(0.000001, 4, priceFormatOptions.AllowDecimalPips, null, 2);
+        expect(parts.Pre).toEqual('');
+        expect(parts.First).toEqual('0.00');
+        expect(parts.Pips).toEqual('00');
+        expect(parts.DeciPips).toEqual('01');
+        expect(parts.Post).toEqual('');
+
+        // When formatAsPips

```

Comment:
Does negative work too?

Reply:
Added a couple of tests to cover negative prices